CHANGELOG.md

```diff
@@ -1,8 +1,33 @@
-# Thrust 1.10.0 (NVIDIA HPC SDK 20.9)
+# Thrust 1.11.0 (NVIDIA HPC SDK 20.11)
```

Comment:
Let's not modify the changelog on a per-commit basis. This will lead to merge conflict hell, and this ended up missing the 1.11.0 release.

If we want to include changelogs in commits, let's create a directory for them so each new entry can be put into it's own markdown file. We used this approach successfully in VTK-m -- then we just merge and remove them when we cut the release.

Reply:
Reverted these changes, but kept the note that 1.10.0 shipped with CTK 11.2.